composer.json

```diff
@@ -75,6 +76,9 @@
             "web/profiles/contrib/{$name}": ["type:drupal-profile"],
             "web/themes/contrib/{$name}": ["type:drupal-theme"],
             "drush/contrib/{$name}": ["type:drupal-drush"]
-        }
+        },
+        "preserve-paths": [
+            "web/sites/development.services.yml"
```

Comment:
Thanks @RobLoach. Actually this is an issue with Drupal Scaffold's "initial" parameter. The defaults are supposed to include `development.services.yml` and `default/services.yml`, but they don't. I made branch for it on FTUSA after talking with Jay, but I'll PR it here instead. Mind rolling back your change here? I don't mind keeping the `preserve-paths` dependency, as it's needed for other stuff too.